update.py

```
@@ -1 +1,135 @@
+    if response.status_code == 200:
+        return response.json()[0]['sha']
+    else:
+        raise RuntimeError("Unable to find the most recent version of the script.")
+
+
+def download_update():
+    """ Download the most recent version of the script and store it as a zip file. """
+    response = requests.get(download_url, timeout=30)
+
+    if response.status_code != 200:
+        raise RuntimeError("Unable to download the most recent version of the script.")
```

Ruff (0.13.3)

Avoid specifying long messages outside the exception class

(TRY003)